ovos_tts_plugin_server/__init__.py

```diff
@@ -56,13 +54,7 @@ def get_tts(
         }
         if not voice or voice == "default":
             params.pop("voice")
-        if self.host:
-            if isinstance(self.host, str):
-                servers: List[str] = [self.host]
-            else:
-                servers = self.host
-        else:
-            servers = self.public_servers
+        servers = self.host or random.shuffle(self.public_servers)
```

Comment:
The second condition will never be reached because `self.host` now always returns a list of strings